PetProject/Screens/GamesList/ViewModel+VC/GamesListViewModel.swift

```diff
@@ -28,29 +30,38 @@ class GamesListViewModel: ObservableObject {
     }
     
     func paginateDealsList() {
+        DispatchQueue.main.async { [weak self] in
```

Comment:
This is not necessary. Try to make the VM methods run in the main queue. Try to stick to this, it will make the code more predictable.

Reply:
Done